.github/workflows/merge-dependabot.yml

```diff
@@ -0,0 +1,15 @@
+
+name: auto-merge
+on:
+  pull_request:
+    types: [ labeled ]
```

Comment:
What is the motivation for this `types`?

Reply:
PRs from dependabot have the `labels` like `dependencies`, etc. This is to filter on only labeled PRs